src/Sulu/Bundle/MediaBundle/Media/FormatLoader/BaseXmlFormatLoader.php

```diff
@@ -24,6 +24,8 @@
 
     const SCHEMA_URI = '';
 
+    const SCHEME_PATH = '';
```

Comment:
normally set in the extended classes. as used in L166 with `static` the correct values from the extended classes is used.